### DISCUSSION TITLE
What is the plan for "tiny differentiable simulator" and "bullet"?

### DISCUSSION THREAD

**Discussion by cadop** [bulletphysics/bullet3#3873] [Q&A] [answered]
Hi @erwincoumans , could you comment on the relationship you see between bullet and tiny differentiable simulator just released? Is the plan to integrate it with bullet, or will you be focusing more on that is parallel and ultimately replace bullet?  Or are they for different applications (and if so, when would you recommend using one over the other)?

Thanks!

**Comment by erwincoumans** [comment, marked as answer]
PyBullet is more general purpose, TDS is more specific for researchers who like to dig into differentiable simulation.
TDS is used in some of our research, for example this accepted ICRA 2021 paper: https://arxiv.org/abs/2011.04217
Aside from researchers, who focus is on robotics and/or reinforcement learning and use those simulators as one of their tools, there are no real teams, or plans or resources behind either project.
